components/appmgt/org.wso2.carbon.appmgt.custom.mediators/pom.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+# Copyright (c) 2005-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
```

Comment:
This PR was created on Apr 8, 2015. At that time we used year range for the license.